File: SoloLearn Codes/Ballpark Orders/ballpark_orders.py
foods = {
    "Nachos": 6,
    "Pizza": 6,
    "Cheeseburger": 10,
    "Water": 4,
    "Coke": 5,
}

items = input().split()
cost = sum(foods[i] if i in foods else foods["Coke"] for i in items)

print(cost + cost * .07)

File: SoloLearn Codes/Ballpark Orders/BallparkOrders.cs
using System;
using System.Collections.Generic;
using System.Linq;


namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            IDictionary<string, int> lst = new Dictionary<string, int>() {
                {"Nachos", 6},
                {"Pizza", 6},
                {"Cheeseburger", 10},
                {"Water", 4},
                {"Coke", 5 }
            };
            string[] s = Console.ReadLine().Split();
            int cost = s.Select(w => lst.ContainsKey(w) ? lst[w] : lst["Coke"]).Sum();
            Console.WriteLine(cost + cost * .07);
        }
    }
}